media/editor/dataDisplay.tsx

```diff
@@ -506,20 +492,20 @@ const DataCell: React.FC<{
 				ctx.isSelecting = undefined;
 			} else if (e.shiftKey && prevFocused) {
 				// on a shift key, the user is expanding the selection (or deselection)
-				// of an existing byte. We *don't* include that byte since we don't want
-				// to swap the byte.
+				// of an existing offset. We *don't* include that offset since we don't want
+				// to swap the offset.
 				if (e.ctrlKey || e.metaKey) {
-					ctx.addSelectionRange(Range.inclusive(prevFocused.byte, byte));
+					ctx.addSelectionRange(Range.inclusive(prevFocused.byte, offset));
 				} else {
-					ctx.setSelectionRanges([Range.inclusive(prevFocused.byte, byte)]);
+					ctx.setSelectionRanges([Range.inclusive(prevFocused.byte, offset)]);
 				}
 			} else if (e.ctrlKey || e.metaKey) {
-				ctx.addSelectionRange(Range.single(byte));
+				ctx.addSelectionRange(Range.single(offset));
 			} else {
-				ctx.setSelectionRanges([Range.single(byte)]);
+				ctx.setSelectionRanges([Range.single(offset)]);
 			}
 		},
-		[focusedElement.key, byte],
+		[focusedElement.key, offset, isAppend],
```

Comment:
isAppend does not seem used here

```suggestion
		[focusedElement.key, offset],
```